genai-perf/genai_perf/inputs/retrievers/synthetic_data_retriever.py

```diff
@@ -62,7 +68,10 @@ def retrieve_data(self) -> List[Dict[str, Any]]:
                     image_height_stddev=self.config.image_height_stddev,
                     image_format=self.config.image_format,
                 )
-                data["image"] = image
+                row.images.append(image)
+            data_rows.append(row)
+        file_name = "synthetic_dataset"
```

Comment:
Do we want to make this a constant somewhere to avoid magic values?

Reply:
I wasn't sure if we needed to unless we wanted to add multi file support to this path as well.
If we do, I will defer that til later on, after we do some more cleaning up.